### PR TITLE
EFRS-954 View images with the extension.tiff.tif

### PR DESCRIPTION
**Pull request by pbohdan**
A new service for uploading images has been created. It contains methods for working with 'jpeg', 'jpg', 'ico', 'png', 'bmp', 'gif', 'tif', 'tiff', 'webp'. To process images with the tiff extension, use the library https://www.npmjs.com/package/tiff.

### CODE REVIEW

File: ui/src/app/core/photo-loader/photo-loader.service.ts [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+import { Injectable } from '@angular/core';
+import { HttpClient } from '@angular/common/http';
+
+import { map, switchMap } from 'rxjs/operators';
+import { Observable } from 'rxjs';
+
+import { decode } from 'tiff';
+import TiffIfd from 'tiff/lib/tiffIfd';
+
+interface IImageData {
```
Comment (isobolev) [inline ui/src/app/core/photo-loader/photo-loader.service.ts:25]: please define the interface in its own file, where other interfaces are placed. Do not use 'I' prefix for interfaces, according to the angular style guide it is redundant

File: ui/src/app/core/photo-loader/photo-loader.service.ts [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+  width: number;
+}
+
+@Injectable({
+  providedIn: 'root',
+})
+export class LoadingPhotoService {
+  constructor(private http: HttpClient) {}
+
+  public tiffConvertor(url: string): Observable<ImageBitmap> {
```
Comment (isobolev) [inline ui/src/app/core/photo-loader/photo-loader.service.ts:37]: please remove public keywords for all methods, according to the angular style guide it is redundant

File: ui/src/app/features/face-services/face-verification/verification-result/verification-result.component.ts [hunk 3]
```diff
@@ -102,13 +100,17 @@ export class VerificationResultComponent implements OnChanges, OnDestroy {
...
+    return this.loadingPhotoService.loader(file).pipe(
+      tap((bitmap: ImageBitmap) => {
+        canvasSize.height = (bitmap.height / bitmap.width) * canvasSize.width;
         canvas.nativeElement.setAttribute('height', canvasSize.height);
+        this.imgCanvas = bitmap;
       }),
       map(imageSize => this.prepareForDraw(imageSize, data, canvasSize, key)),
-      map(preparedImageData => this.drawCanvas(canvas, preparedImageData, file, canvasSize))
+      map(preparedImageData => {
+        console.log();
```
Comment (isobolev) [inline ui/src/app/features/face-services/face-verification/verification-result/verification-result.component.ts:111]: please remove console.log

**Review by isobolev — COMMENTED** [review]
looks good, couple comments.

**Comment by isobolev** [comment]
@endypanda you are more familiar with this area, please take a look

**Review by isobolev — APPROVED** [review]

**Review by endypanda — APPROVED** [review]

**Review by isobolev — APPROVED** [review]

**Pull request merged**